src/RateLimiting/src/TokenBucketRateLimiter.cs

```diff
@@ -4,16 +4,19 @@
 // Pending dotnet API review
 
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Threading.Tasks;
 
 namespace System.Threading.RateLimiting
 {
-#pragma warning disable 1591
+    /// <summary>
+    /// <see cref="RateLimiter"/> implementation that replenishes tokens periodically instead of via a release mechanism.
+    /// </summary>
     public sealed class TokenBucketRateLimiter : RateLimiter
     {
         private int _tokenCount;
         private int _queueCount;
-        private long _lastReplenishmentTick;
+        private long _lastReplenishmentTick = Environment.TickCount;
```

Comment:
```suggestion
        private long _lastReplenishmentTick = Environment.TickCount * TimeSpan.TicksPerMillisecond;
```

Though I think we should just use `Environment.TickCount64` and not multiply by 10,000 everywhere.